eng/common/templates/jobs/source-build.yml

```diff
@@ -14,7 +14,7 @@ parameters:
   # This is the default platform provided by Arcade, intended for use by a managed-only repo.
   defaultManagedPlatform:
     name: 'Managed'
-    container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream8-20220809204800-17a4aab'
+    container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream8'
```

Comment:
@hoyosjs should I do something about this change? Do I need to revert it?

Reply:
```suggestion
    container: 'mcr.microsoft.com/dotnet-buildtools/prereqs:centos-stream8-20220809204800-17a4aab'
```

Reply:
Same here. This is a "eng/common" change which must be done in arcade.